src/lib/update-alternatives/UI/main_dialog.rb

```diff
@@ -33,26 +33,35 @@ def dialog_options
     def dialog_content
       VBox(
         create_table,
-        HBox(
-          PushButton(Id(:show), _("Show alternatives")),
-          PushButton(Id(:cancel), _("Cancel"))
-        )
+        footer
       )
     end
 
     def show_handler
       AlternativesDialog.new.run
     end
 
+    def alternatives_table_handler
+      AlternativesDialog.new.run
+    end
+
     def create_table
       Table(
         Id(:alternatives_table),
-        Header(_("Name"), _("Actual alternative")),
+        Opt(:notify),
+        Header(_("Name"), _("Actual alternative"), _("Status")),
         [
-          Item(Id(:java), "java", "/usr/lib64/jvm/jre-1.8.0-openjdk/bin/java"),
-          Item(Id(:an_alternative_name), "an alternative name", "Alternative_path")
+          Item(Id(:java), "java", "/usr/lib64/jvm/jre-1.8.0-openjdk/bin/java", _("auto")),
+          Item(Id(:an_alternative_name), "an alternative name", "Alternative_path", _("manual"))
```

Comment:
Do you plan to use manual or better specify there which one is selected like `vim`?